libs/langgraph/src/errors.ts

```diff
@@ -18,6 +18,12 @@ export class BaseLangGraphError extends Error {
   }
 }
 
+export class GraphBubbleUp extends BaseLangGraphError {
+  get is_bubble_up() {
```

Comment:
nit, camel case?

Reply:
Ah I was following the names you have elsewhere in this file?